### PR TITLE
Initial recipe

### PR DESCRIPTION
**Pull request by cdeepali**
Adaptation of `conda-forge` feedstock - https://github.com/conda-forge/pytorch-lightning-feedstock

### CODE REVIEW

**Comment by cdeepali** [comment]
I have removed `pip check` from the test section because it failed with the following error:


```
+ pip check
torchtext 0.8.0a0+c851c3e requires sentencepiece, which is not installed.
onnx 1.6.0 requires typing-extensions, which is not installed.
tensorflow 2.3.0 has requirement keras-preprocessing<1.2,>=1.1.1, but you have keras-preprocessing 1.1.0.
pytorch-lightning 0.9.0 has requirement tensorboard==2.2.0, but you have tensorboard 2.3.0.
pytorch-lightning 0.9.0 has requirement tqdm>=4.41.0, but you have tqdm 4.36.1.
```

**Comment by cdeepali** [comment]
I am checking if other packages tolerate `tqdm `- `4.41`*

**Comment by cdeepali** [comment]
This is building with our current `tqdm` version too, however the requirements specify `tqdm>=4.41.0` . I verified building other packages that depend on `tqdm`, so I think we can update tqdm pin too.

**Review by jayfurmanek — APPROVED** [review]
OK, great. Thanks @cdeepali

**Comment by cdeepali** [comment]
@jayfurmanek, I added `torchvision-base` as a dependency too here. Request you to approve once again.

File: README.md [hunk 1]
```diff
@@ -1 +1,436 @@
-# pytorch-lightning-feedstock
\ No newline at end of file
+<div align="center">
```
Comment (bnemanich) [inline README.md:1]: Don't add this README from pytorch lightning. Eventually we will have OpenCE README files.

Comment (cdeepali) [reply README.md:1]: yeah it is copy of upstream - https://raw.githubusercontent.com/PyTorchLightning/pytorch-lightning/0.9.0/README.md. 
Shall I remove it?

Comment (cdeepali) [reply README.md:1]: re-read, I will remove it.

**Comment by cdeepali** [comment]
Removed the README content.

**Comment by jayfurmanek** [comment]
The pip check results above are interesting. That may need a bit of looking into.

**Comment by cdeepali** [comment]
So shall we investigate first and then add this recipe. Please advise.

**Comment by jayfurmanek** [comment]
I think let's go ahead and get this one in. The tqdm is being addressed with:
https://github.com/open-ce/open-ce/pull/14

The others are potential issues in elsewhere.

**Review by jayfurmanek — APPROVED** [review]

**Comment by cdeepali** [comment]
Sure, I will merge. Thanks @jayfurmanek.

**Pull request merged**